docs/en/logs/using.asciidoc

```diff
@@ -0,0 +1,97 @@
+[role="xpack"]
+[[xpack-logs-filtering]]
+=== Explore and filter logs data
+Use the {logs-app} in {kib} to explore and filter your logs in real time.
+
+You can customize the output to focus on the data you want to see and to control how you see it.
+You can also view related application traces or uptime information where available.
+
+[role="screenshot"]
+image::images/logs-console.png[Logs Console in Kibana]
+
+[float]
+[[logs-search]]
+==== Use the power of search
+
+To perform ad hoc searches for specific text, use the search bar.
+You can also create structured queries using {kibana-ref}/kuery-query.html[Kibana Query Language].
+For example, enter `host.hostname : "host1"` to see only the information for `host1`.
+// ++ this isn't quite the same as the corresponding metrics description now.
+
+[float]
+[[logs-configure-source]]
+==== Configure the data to use for your logs
+Are you using a custom index pattern to store the log entries?
+Do you want to limit the entries shown or change the fields displayed in the columns?
+If so, <<xpack-logs-configuring, configure the logs source data>> to change the index pattern and other settings.
+
+[float]
+[[logs-time]]
+==== Specify the time and date
+
+Click image:images/time-filter-calendar.png[time filter calendar], then choose the time range for the logs.
+
+Log entries for the specified time appear in the middle of the page. To quickly jump to a nearby point in time, click the minimap timeline to the right.
+// ++ what's this thing called? It's minimap in the UI. Would timeline be better?
```

Comment:
Can we get rid of all of these comments? I think they're just the personal notes of an old Obs writer